app/lib/actions/katello/organization/manifest_refresh.rb

```diff
@@ -70,12 +70,7 @@ def humanized_name
         # results in correct grammar on Tasks page,
         # e.g. "Refresh manifest for organization Default Organization"
         def humanized_input
-          [
-            [:organization, {
-              :text => "for organization '#{input[:organization_name]}'",
-              :link => "/organizations/#{input[:organization_id]}/edit"
-            }]
-          ]
+          "for organization '#{input[:organization_name]}'"
```

Comment:
Changed to returning just a string here.  For some reason it wasn't working before, but I think it was just because I hadn't restarted the server or something.